host-os-iso-creator.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) IBM Corp. 2016.
```

Comment:
About my backwards compatibility concerns, @maurorodrigues explained to me that this is a problem only to the versions repository, so they may be discarded here.
I keep my suggestion of naming to create_host_os_iso.py and (unrelated to this PR) create_host_os_build.py or create_host_os_packages.py